pkg/templates/scripts/mirror/imageset.yaml.template

```diff
@@ -14,3 +14,5 @@ mirror:
 {{.AdditionalImages}}{{end}}
 {{if .BlockedImages}}  blockedImages:
 {{.BlockedImages}}{{end}}
+{{if .Operators}}  operators:
```

Comment:
we need to make should this is empty and with a valid format when there are not operators at all.

Reply:
Yeah, ensured again that it's indeed working.